lldb/include/lldb/Target/Process.h

```diff
@@ -2685,6 +2685,13 @@ void PruneThreadPlans();
   lldb::addr_t FindInMemory(lldb::addr_t low, lldb::addr_t high,
                             const uint8_t *buf, size_t size);
 
+  Status FindInMemory(AddressRanges &matches, const uint8_t *buf, size_t size,
+                      const AddressRanges &ranges, size_t alignment,
+                      size_t max_count);
```

Comment:
It would be nice to keep the `lldb_private::Process` and the `lldb::SBProcess` APIs similar in return values and argument ordering.